sales/send_quotations/terms_and_conditions.rst

```diff
@@ -0,0 +1,41 @@
+================================
```

Comment:
Your page isn't included in any toctree. You could see it in the build logs.

/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getsignaturetovalidate.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/invoice_and_deliver_to_different_address.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/quotationsdeadline.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:: WARNING: document isn't included in any toctree

These errors aren't critical, but the page won't be available if you don't know the exact URL. 

![screenshot from 2018-05-17 16-18-24](https://user-images.githubusercontent.com/8435879/40183411-14fdc820-59ee-11e8-8f70-0aca2f40c7dc.png)